src/Workers/Worker.ts

```diff
@@ -43,25 +43,24 @@ const loadTrackables = async (msg: any) => {
     let src = msg.data;
     let refRows = msg.trackableHeight;
     let refCols = msg.trackableWidth;
-    //let mat = new cv.matFromArray(refRows, refCols, cv.CV_8UC4, src);
-    //var mat = new cv.Mat(imageData.height, imageData.width, cv.CV_8UC4);
+
     let mat = new cv.Mat(refRows, refCols, cv.CV_8UC4);
-    //mat.data.set(imageData.data);
+
     mat.data.set(src.data);
     console.log(mat);
 
-    cv.cvtColor(mat, mat, cv.COLOR_RGBA2GRAY, 0);
+    cv.cvtColor(mat, mat, cv.COLOR_RGBA2GRAY);
 
     let ksize = new cv.Size(BlurSize, BlurSize);
     let anchor = new cv.Point(-1, -1);
-    cv.blur(mat, mat, ksize, anchor, cv.BORDER_DEFAULT);
+    //cv.blur(mat, mat, ksize, anchor, cv.BORDER_DEFAULT);
```

Comment:
Adding blur make more difficult to detect, i will leave this, maybe i need to adjust the settings.